srcpkgs/zita-resampler/template

```diff
@@ -1,22 +1,25 @@
 # Template file for 'zita-resampler'
 pkgname=zita-resampler
-version=1.6.2
+version=1.8.0
 revision=1
+build_wrksrc="source"
 build_style=gnu-makefile
 make_install_args="LIBDIR=/usr/lib"
 hostmakedepends="pkg-config"
-build_wrksrc="source"
-short_desc="A library for resampling audio signals"
+short_desc="Library for resampling audio signals"
 maintainer="silvernode <mollusk@homebutter.com>"
 license="GPL-3.0-or-later"
 homepage="http://kokkinizita.linuxaudio.org/linuxaudio/zita-resampler/resampler.html"
 distfiles="http://kokkinizita.linuxaudio.org/linuxaudio/downloads/${pkgname}-${version}.tar.bz2"
-checksum=233baefee297094514bfc9063e47f848e8138dc7c959d9cd957b36019b98c5d7
+checksum=e5744f23c54dd15b3f783a687bd879eee2a690a4545a15b49c4cf037aa464aa2
 
-CXXFLAGS="-fPIC -DPIC"
+CXXFLAGS="-fPIC -ffast-math"
 
 pre_build() {
-	sed -i '/march=native/d' Makefile
+	case "${XBPS_TARGET_MACHINE}" in
+		x86_64*) ;;
+		*) vsed -i '/DENABLE_SSE2/d' Makefile ;;
```

Comment:
IMO you can depend on SSE2: https://github.com/void-linux/void-packages/pull/28825#issuecomment-781502744

Reply:
oh I didn't know that :D